src/deprecated.jl

```diff
@@ -357,3 +357,32 @@ function by(d::AbstractDataFrame, cols::Any; sort::Bool=false, skipmissing::Bool
 end
 
 @deprecate groupvars(gd::GroupedDataFrame) groupcols(gd)
+
+@deprecate aggregate(d::AbstractDataFrame, fs::Any; sort::Bool=false) begin
+    df = select(d, names(d) .=> [fs])
+    sort && sort!(df)
+    return df
+end
+
+@deprecate aggregate(d::AbstractDataFrame, fs::AbstractVector; sort::Bool=false) begin
+    df = hcat([select(d, names(d) .=> [f]) for f in fs]..., makeunique=true)
+    sort && sort!(df)
+    return df
+end
+
+@deprecate aggregate(gd::GroupedDataFrame, f::Any; sort::Bool=false) begin
+    df = combine(gd, valuecols(gd) .=> [f])
+    sort && sort!(df, setdiff(names(df), groupcols(gd)))
+    return df
+end
+
+@deprecate aggregate(gd::GroupedDataFrame, fs::AbstractVector; sort::Bool=false) begin
+    df = hcat([combine(gd, valuecols(gd) .=> [f], keepkeys=i==1) for (i, f) in enumerate(fs)]...,
+                       makeunique=true)
+    sort && sort!(df, setdiff(names(df), groupcols(gd)))
+    return df
+end
+
+@deprecate aggregate(d::AbstractDataFrame, cols, fs::Any;
+          sort::Bool=false, skipmissing::Bool=false) aggregate(groupby(d, cols, sort=sort,
```

Comment:
Better show the final syntax, otherwise people will get two deprecation warnings, and one is already scary enough. :-)

Reply:
OK